apps/examples/elf/micomapp/Makefile

```diff
@@ -90,6 +90,8 @@ BIN_TYPE = ELF
 DYNAMIC_RAM_SIZE = 20480
 BIN_VER = 20190421
 KERNEL_VER = 2.0
+BIN_STACKSIZE = 2048
+BIN_PRIORITY = 220
```

Comment:
I think it is better to have a config macro for both of this, so that it can be configured at time of compilation only, in default value we can have desired value there